pkg/rehearse/rehearse.go

```diff
@@ -550,6 +550,14 @@ func setupDependencies(
 	}
 
 	g, ctx := errgroup.WithContext(context.Background())
+	g.Go(func() error {
+		appCIConfig := configs[appCIContextName]
+		if err := ensureISTSInQCI(ctx, &appCIConfig, dryRun, requiredImageStreamTags, mirrorOptions, quayIOImageHelper, log); err != nil {
+			// best effort as the required image might be there (just stale) on QCI already and rehearsal can still run with it
+			log.WithError(err).Errorf("failed to ensure imagestreamtags %s in QCI", requiredImageStreamTags)
+		}
+		return nil
+	})
```

Comment:
I ran it on the alpha deployment and it is not working like this. It doesn't have the permission to do so. If we want to do so, we need to add some permission, not sure which one. 
`"error":"[failed to create pairs for build-cache/openshift-ci-tools:master to mirror: failed to get imageStreamTag build-cache/openshift-ci-tools:master from registry cluster: imagestreamtags.image.openshift.io \"openshift-ci-tools:master\" is forbidden: User \"system:serviceaccount:ci:pj-rehearse\" cannot get resource \"imagestreamtags\" in API group \"image.openshift.io\" in the namespace \"build-cache\", failed to create pairs for prucek/golang:1.22 to mirror: failed to get imageStreamTag prucek/golang:1.22 from registry cluster: imagestreamtags.image.openshift.io \"golang:1.22\" is forbidden: User \"system:serviceaccount:ci:pj-rehearse\" cannot get resource \"imagestreamtags\" in API group \"image.openshift.io\" in the namespace \"prucek\"]"`